default-gecko-config

```diff
@@ -15,8 +15,39 @@ else
 ac_add_options --target=arm-linux-androideabi
 fi
 
+# Follow the section 'Bootstrap dependencies' at
+#
+#  https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Simple_Firefox_for_Android_build
+#
+# for installing the Android NDK. Choose 'Firefox for Android' at the
+# prompt. Alternatively, you can also download the NDK manually and set
+# the shell variable B2G_ANDROID_NDK_PATH to the NDK's directory.
+
+if [ "${B2G_ANDROID_NDK_PATH:-0}" != "0" ]; then
+ac_add_options --with-android-ndk="$B2G_ANDROID_NDK_PATH"
+else
+ac_add_options --with-android-ndk="$HOME/.mozbuild/android-ndk-r11b"
+fi
```

Comment:
nit: we could just have:

B2G_ANDROID_NDK_PATH=${B2G_ANDROID_NDK_PATH:="$HOME/.mozbuild/android-ndk-r11b"}
ac_add_options --with-android-ndk="${B2G_ANDROID_NDK_PATH}"

instead of the if/else